docs(contracts): update comments

satello opened the pull request:
https://docs.google.com/document/d/1MOyG_YdKujdfXAqzhjJKOPnuZpXQo6yD8FB9Ryrl4kM/edit?usp=sharing

- Format event logs so that all long function declarations are broken up onto separate lines (instead of only some)
- Update comments for grammatical errors and to make them clearer

clesaege commented:
The code at l355 was correct and the proposed modification is incorrect as it would prevent funding during the second half of the period in case of tie.

satello commented:
My mistake. I had changed that in the first pass but realized that in the second but apparently forgot to revert the change. Updated now, and there should be no changes to the logic.

mtsalenc commented:
Looks good. I'm merging this and adding the modifications suggested in the doc on another commit.
